### ISSUE TITLE
Fresh Install of Game Crashes Immediately Upon Launch

### ISSUE THREAD

**Issue by owentroe** [anegostudios/VintageStory-Issues#1303]
**Game Version:** 1.15.5
**Platform:** Windows 7
**Modded:** No

### Description
After fresh install, game crashes immediately upon launch. 

### Logs
```
Running on 64 bit Windows with 8 GB RAM 
Version: v1.15.5 (Stable)9/17/2021 11:35:53 PM: Critical error occurred
System.MissingMethodException: Method not found: '!!0[] System.Array.Empty()'.
   at _q065sSmA3CNgGuKTGnlAjR4ljPw._WkSl7yfiyr5DwPiSMvTudNAhp6(_t3ILnyYBq5OoYhhBY3JeTWgFmJDA , String[] )
   at _DBu5wiQWRAHGS5G2jiCKymvQ9yE._WkSl7yfiyr5DwPiSMvTudNAhp6(ThreadStart )
-------------------------------

```

**Comment by copygirl** [comment]
I believe this should be fixed by installing .NET Framework 4.6.1 or 4.6.2.
Go [here](https://dotnet.microsoft.com/download/dotnet-framework/net462) and download the Runtime for this version of .NET.

**Comment by copygirl** [comment]
Going to assume this is fixed, as it also worked in #1186.

**Comment by owentroe** [comment]
Just tested it now, it works. Thanks for your help.
